pkg/spdk/engine.go

```diff
@@ -570,7 +569,6 @@ func (e *Engine) ValidateAndUpdate(spdkClient *spdkclient.Client) (err error) {
 	// Verify engine map consistency
 	for replicaName := range e.ReplicaAddressMap {
 		if _, exists := e.ReplicaBdevNameMap[replicaName]; !exists {
-			e.ReplicaBdevNameMap[replicaName] = ""
```

Comment:
I am not sure if this modification is OK since I want to make sure all maps hold the same key set after the validation.